packages/epo-widget-lib/src/widgets/LightCurvePlot/LightCurve/index.tsx

```diff
@@ -0,0 +1,46 @@
+import { FunctionComponent } from "react";
+import { range } from "d3-array";
+import { Domain, ScaleFunction } from "@/charts/types";
+import { estimateMagnitude } from "../helpers";
+
+interface LightCurveProps {
+  xScale: ScaleFunction;
+  yScale: ScaleFunction;
+  yOffset: number;
+  gaussianWidth: number;
+  xDomain: Domain;
+  yDomain: Domain;
+}
+
+const LightCurve: FunctionComponent<LightCurveProps> = ({
+  xScale,
+  xDomain,
+  yScale,
+  yDomain,
+  gaussianWidth,
+  yOffset,
+}) => {
+  const magnitudes = range(xDomain[0], xDomain[1], 0.5).map((x) => {
+    return { x, y: estimateMagnitude(x, gaussianWidth) };
+  });
+
+  const curve = magnitudes
```

Comment:
use a path so we can re-render using the same number of points so it can be animated if we want (probably not), can also take advantage of yOffset being a simple translation and do that via `transform` instead of recalculating